Library/Homebrew/cmd/update.sh

```diff
@@ -352,6 +373,11 @@ EOS
         exit
       fi
 
+      if [[ -n "$HOMEBREW_VERBOSE" ]]
+      then
+        echo "Fetching $DIR..."
+      fi
+      
```

Comment:
I think we can skip this. Feels like the preceding “Checking for Homebrew updates...” is enough to make it clear to the user what is going on and this will just make the output unwieldy for a big number of taps (though only the taps with pending updates will trigger this) and may cause overlap in the output due to the parallel nature of this code section.